src/element-observation.js

```diff
@@ -70,17 +70,24 @@ export class StyleObserver {
     this.element.style.setProperty(style, value, priority);
   }
 
+  _fixCase(m) {
+    return '-' + m.toLowerCase();
+  }
+
   setValue(newValue) {
     let styles = this.styles || {};
     let style;
     let version = this.version;
 
     if (newValue !== null && newValue !== undefined) {
       if (newValue instanceof Object) {
+        let value;
         for (style in newValue) {
           if (newValue.hasOwnProperty(style)) {
+            value = newValue[style];
+            style = style.replace(/([A-Z])/g, this._fixCase);
```

Comment:
@jdanyow  Yeah, that's fine. I had originally done that but I wasn't sure and thought closures and the like were discouraged.